pkg/controller/servicemesh/memberroll/controller_test.go

```diff
@@ -464,6 +464,33 @@ func TestReconcileNamespacesIgnoresControlPlaneNamespace(t *testing.T) {
 	assert.DeepEquals(configuredMembers, []string{appNamespace}, "reconcileNamespaces returned an unexpected configuredMembers list", t)
 }
 
+func TestReconcileWorksWithMultipleNamespaces(t *testing.T) {
+	roll := newDefaultMemberRoll()
+	addOwnerReference(roll)
+	roll.Spec.Members = []string{appNamespace, appNamespace2}
+	roll.ObjectMeta.Generation = 2
+	roll.Status.ObservedGeneration = 1
+	controlPlane := markControlPlaneReconciled(newControlPlane(""), meshVersionDefault, operatorVersionDefault)
+	roll.Status.ServiceMeshGeneration = controlPlane.Status.ObservedGeneration
+	roll.Status.MeshVersion = controlPlane.Status.AppliedVersion
+	namespace := &core.Namespace{
+		ObjectMeta: meta.ObjectMeta{
+			Name: controlPlaneNamespace,
+		},
+	}
+
+	cl, _, r, _, kialiReconciler := createClientAndReconciler(t, roll, controlPlane, namespace, newNamespace(appNamespace))
+	assertReconcileSucceeds(r, t)
+	cl.Create(context.TODO(), newNamespace(appNamespace2))
```

Comment:
Unhandled error. While the error itself is probably never actually returned because we're using the fake client, IMHO it's always good to handle errors. If nothing else, the static analysis tools won't complain about unhandled errors.

I have another PR I'm about to submit, where I introduce `test.SuccessOrPanic()`, which, in the above case, would be used like this: 
```
test.SuccessOrPanic(cl.Create(context.TODO(), newNamespace(appNamespace2))
```

Reply:
I'm using `test.PanicOnError()` now